### PR TITLE
refactor(console): update loading skeletons

### PR DESCRIPTION
**Pull request by xiaoyijun**
## Summary

Update loading skeletons according to new AC layouts.

### Before
image

### After
image

### Before
image

### After
image

### CODE REVIEW

File: packages/console/src/components/DetailsSkeleton/index.module.scss [hunk 1]
```diff
@@ -46,42 +50,73 @@
...
+    padding: _.unit(6);
 
-    .tabBar {
-      @include _.shimmering-animation;
-      width: 100%;
-      height: 25px;
-      margin-bottom: _.unit(13.5);
-    }
+    .introduction {
+      width: 296px;
```
Comment (simeng-li) [inline packages/console/src/components/DetailsSkeleton/index.module.scss:68]: just curious about those magic numbers, should they be aligned with component width or just a random 296 from figma?

File: packages/console/src/pages/SignInExperience/components/Skeleton/index.module.scss [hunk 2]
```diff
@@ -2,145 +2,146 @@
...
-          width: 100%;
-          height: 36px;
-          margin-top: _.unit(3);
-          @include _.shimmering-animation;
+        background-color: var(--color-layer-1);
+        border-radius: 16px 16px 0 0;
+
+        .info {
+          .title {
+            width: 113px;
```
Comment (simeng-li) [inline packages/console/src/pages/SignInExperience/components/Skeleton/index.module.scss:70]: ditto and for all the rest hard coded width number bellow

File: packages/console/src/components/DetailsSkeleton/index.module.scss (hunk 1 above)
Comment (xiaoyijun) [reply packages/console/src/components/DetailsSkeleton/index.module.scss:68]: This follows the design settings.
image

File: packages/console/src/pages/SignInExperience/components/Skeleton/index.module.scss (hunk 2 above)
Comment (xiaoyijun) [reply packages/console/src/pages/SignInExperience/components/Skeleton/index.module.scss:70]: I think it's not a big deal, we don't change its size, just adjust the code position.

Comment (xiaoyijun) [reply packages/console/src/pages/SignInExperience/components/Skeleton/index.module.scss:70]: https://www.figma.com/file/iroUuWbeZ9fqcqXNwqYZai/%F0%9F%92%BB-%5BAC%5D-Admin-Console-V1-Master?node-id=6581%3A231829&t=9Qdo9hQWpBvVXlZE-4

**Review by charIeszhao — APPROVED** [review]

**Comment by charIeszhao** [comment]
Let's go!

**Pull request merged**